ajax/libs/react-uwp/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "react-uwp",
+  "description": "UWP Design & Fluent Design UiKit by React",
+  "filename": "css/segoe-mdl2-assets.css",
+  "keywords": [
+    "uwp",
+    "microsoft-uwp",
+    "fluent",
+    "microsoft-fluent",
+    "uwp-design",
+    "fluent-design",
+    "react",
+    "typescript",
+    "javascript",
+    "react-components"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/myxvisual/react-uwp.git"
+  },
+  "author": "myxvisual",
+  "license": "MIT",
+  "homepage": "https://www.react-uwp.com",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/myxvisual/react-uwp-bower.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "**/*.js",
+          "**/*.css",
+          "**/*.svg",
+          "**/*.ttf",
+          "**/*.woff",
+          "**/*.woff2",
+          "**/*.eot"
```

Comment:
It's useful to us to let the code clear, would you modify from A to B?
- A
```
          "**/*.js",
          "**/*.css",
          "**/*.svg",
          "**/*.ttf",
          "**/*.woff",
          "**/*.woff2",
          "**/*.eot"
```

- B
```
          "**/*.+(js|css|svg|ttf|woff|woff2|eot)"
```
(This matches one or more occurences of the patterns provided.)

And do you want to add `*png` files?